### PR TITLE
Reordered SASS to remove linting warnings

### PR DESCRIPTION
**Pull request by Lukewh**
# Done

Fixes [my own sanity](http://lh5.googleusercontent.com/-T5gHuO5DlUw/TXWCD_ao89I/AAAAAAAAAjo/jIXW7jmCjbk/Tom-Cruise-laughing.jpg).

- Reordered scss properties to be alphabetical

### CODE REVIEW

File: static/sass/_patterns_inline-list--compact.scss [hunk 1]
```diff
@@ -1,6 +1,8 @@
 @mixin p-inline-list--compact {
   .p-inline-list--compact {
+    // sass-lint:disable placeholder-in-extend
```
Comment (bartaz) [inline static/sass/_patterns_inline-list--compact.scss]: Shouldn't `// sass-lint:disable-line ...` work just fine for one-liners like this?

File: static/sass/_patterns_maas_modal.scss [hunk 2]
```diff
@@ -104,7 +104,13 @@ $color-modal-controls: rgba($color-x-dark, .2);
 
   .vbox-next,
   .vbox-prev {
-    background-color: $color-x-light;
+    background: {
```
Comment (bartaz) [inline static/sass/_patterns_maas_modal.scss]: This file was a just straight copy from maas, so didn't want to touch it.
But if it helps your sanity... ;)

File: static/sass/_patterns_inline-list--compact.scss (hunk 1 above)
Comment (Lukewh) [reply static/sass/_patterns_inline-list--compact.scss]: Good catch, will fix

File: static/sass/_patterns_maas_modal.scss (hunk 2 above)
Comment (Lukewh) [reply static/sass/_patterns_maas_modal.scss]: `./run test` all those lines.....

**Pull request merged**